src/ol/format/kml.js

```diff
@@ -1078,6 +1108,9 @@ ol.format.KML.readStyle_ = function(node, objectStack) {
   var imageStyle = /** @type {ol.style.Image} */
       ('imageStyle' in styleObject ?
           styleObject['imageStyle'] : ol.format.KML.DEFAULT_IMAGE_STYLE_);
+  if (imageStyle == ol.format.KML.DEFAULT_NO_IMAGE_STYLE_) {
```

Comment:
Thanks for the explanation. In this case I'd opt for `null` - smaller build size and the name of the constant has all information that's needed when reading the code.